components/Card/index.js

```diff
@@ -1,16 +1,40 @@
 import styled from "styled-components";
+import Rating from "../Rating";
+import AverageRating from "../AverageRating";
+import { useState, useEffect } from "react";
+
+export default function Card({ card, mutateCards }) {
+  const [rating, setRating] = useState(0);
+
+  useEffect(() => {
+    setRating(0);
+  }, [card.id]);
+
+  async function handleRating(score) {
+    const url = `/api/submitRating?id=${card.id}&rating=${score}`;
```

Comment:
I would recommend to provide the data in the request body rather than in the arguments.